ksqldb-tools/src/test/java/io/confluent/ksql/tools/migrations/MigrationsTest.java

```diff
@@ -199,18 +221,35 @@ public static void setUpClass() throws Exception {
   @AfterClass
   public static void classTearDown() {
     CONNECT.shutdown();
-    REST_APP.getPersistentQueries().forEach(str -> makeKsqlRequest("TERMINATE " + str + ";"));
+    REST_APP.closePersistentQueries();
   }
 
   @Before
-  public void setUp() {
+  public void setUp() throws Exception {
+    if (withMigrationsDirOverride) {
+      migrationsDir = Paths.get(testDir, "my/custom/migrations_dir").toString();
+      assertThat(new File(migrationsDir).mkdirs(), is(true));
```

Comment:
Does the migration directory have to exist in order to be used?

Reply:
Yes, it does.